rules/cross-platform/external_alerts.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/07/08"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/08"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an external alert is received. Enabling this rule allows you to immediately being
+investigating external alerts in the app.
+"""
+language = "kuery"
+license = "Elastic License"
+name = "External Alerts"
+risk_score = 47
+rule_id = "eb079c62-4481-4d6e-9643-3ca499df7aaa"
+rule_name_override = "message"
+severity = "medium"
+tags = ["Elastic"]
+timestamp_override = "event.ingested"
```

Comment:
I don't think we want to override the timestamp here since `event.ingested` might not be filled in for most external alerts?

Reply:
when a timestamp override is defined and missing, does it default back to `@timestamp`?

Reply:
Yes, will need to add to signal metadata so the user knows this happened.